packaging/Makefile

```diff
@@ -34,9 +34,9 @@ docker-build: docker.build.amd64 docker.build.arm64 docker.build.armv7
 docker.build.amd64: clean skipper eskip
 	docker build -t $(IMAGE) .
 docker.build.arm64: clean build.linux.arm64 docker.build.enable
-	docker build build -t $(ARM64_IMAGE) --platform linux/arm64 -f Dockerfile.arm64 .
+	docker build -t $(ARM64_IMAGE) --platform linux/arm64 -f Dockerfile.arm64 .
```

Comment:
`docker buildx build`

Reply:
sorry I did the  `:%s/buildx/build/g`, which is wrong.
`docker buildx build` is correct.